src/components/body/Body.ts

```diff
@@ -12,8 +12,7 @@ import { Subscription } from 'rxjs/Subscription';
   selector: 'datatable-body',
   template: `
     <div>
-      <datatable-progress
```

Comment:
I actually prefer this pattern, similar how React devs do attributes. Any reasons why you did this?